index.html

```diff
@@ -41,5 +41,19 @@
     </a>
 </div>
 </div>
-    </body>
+
+<div class="ui divider"></div>
+<div class="ui grid">
+    <div class="five wide column">
```

Comment:
Would you mind changing this to 4, IMO it looks better with an even number :laughing: